CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.9.0 (TBD)
+
+BREAKING CHANGES:
+ - [types] ResponseEndBlock: renamed Diffs field to ValidatorUpdates
```

Comment:
also the field indices for the oneof Request and Response types

Reply:
yes.